src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString
+    .split(';')
+    .filter((style) => style.trim())
+    .map((style) => {
+      const [property, value] = style.split(':');
+
+      return [property.trim(), value.trim()];
+    })
+    .reduce((acc, [property, value]) => {
+      acc[property] = value;
+
+      return acc;
+    }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to improve readability.

Consider refactoring your code to match the good example provided in the checklist:

const cssStyles = styles
 .map(style => style)
 .filter(style => style.length)
 .slice(0, 5)